tests/run-tests

```diff
@@ -2,8 +2,4 @@
 set -e
 
 cd "$(dirname "$0")"
-if pytest=$(which py.test-3); then
-    exec "$pytest" "$@"
-else
-    exec python3 -m unittest discover --top-level-directory .. "$@"
-fi
+exec pytest "$@"
```

Comment:
I think you need to spell this as [`pytest-3` or `py.test-3`](https://packages.debian.org/bullseye/all/python3-pytest/filelist):

```
../../../tests/run-tests: line 5: exec: pytest: not found
```

Reply:
Ugh, I thought I checked that. I switched it to `python3 -m pytest` since that will work universally.